docs/get_started/centos_setup.md

```diff
@@ -0,0 +1,164 @@
+# Installing MXNet on CentOS
+MXNet currently supports Python, R, Julia, and Scala. For users on CentOS, MXNet provides [a docker installation guide](http://mxnet.io/get_started/docker_setup.html). If you don't have docker environment, please install MXNet step by step.
+
+Note: If you are not familiar with CentOS or building progress, we encourage you to [install or deploy MXNet environment on Ubuntu](http://mxnet.io/get_started/ubuntu_setup.html). 
+
+## Minimum Requirements
+Make sure you have the root permission, and `yum` is properly installed.
```

Comment:
A line about how to know 'yum' is properly installed would be helpful. Since we are asking users to make sure about it.